src/stronghold/mod.rs

```diff
@@ -270,9 +289,7 @@ impl StrongholdAdapter {
         }
```

Comment:
keys_to_re_encrypt should be removed now and instead get the keys from stronghold itself, a method for that was added here https://github.com/iotaledger/stronghold.rs/pull/359

Reply:
Done